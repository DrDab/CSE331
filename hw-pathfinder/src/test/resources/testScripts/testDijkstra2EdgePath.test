# Test if graph with invalid destination node produces correct output.

# Create graph g
CreateGraph g

# Add city names to graph g
AddNode g LosSantos
AddNode g SanAndreas
AddNode g SanFierro
AddNode g LasVenturas
AddNode g Ludendorff
AddNode g Vice
AddNode g Liberty
AddNode g Alderney

# Add edges to graph g
AddEdge g LosSantos SanAndreas 1.23
AddEdge g SanAndreas Vice 65.4326

# Test pathfinding
FindPath g LosSantos Vice
