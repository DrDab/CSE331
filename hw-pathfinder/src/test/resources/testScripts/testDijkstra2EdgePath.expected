# Test if Dijkstra implementation provides correct output navigating 2-edge graph

# Create graph g
created graph g

# Add city names to graph g
added node LosSantos to g
added node SanAndreas to g
added node SanFierro to g
added node LasVenturas to g
added node Ludendorff to g
added node Vice to g
added node Liberty to g
added node Alderney to g

# Add edges to graph g
added edge 1.230 from LosSantos to SanAndreas in g
added edge 65.433 from SanAndreas to Vice in g

# Test pathfinding
path from LosSantos to Vice:
LosSantos to SanAndreas with weight 1.230
SanAndreas to Vice with weight 65.433
total cost: 66.663